src/torcontrol.cpp

```diff
@@ -481,6 +483,53 @@ TorController::~TorController()
     }
 }
 
+void TorController::get_socks_cb(TorControlConnection& _conn, const TorControlReply& reply)
```

Comment:
Make sure parameter names match between declaration and definition :-)

Reply:
I don't agree with that aesthetic.